BackofficeBundle/Resources/public/coffee/backboneRouter.coffee

```diff
@@ -21,8 +21,22 @@ OrchestraBORouter = Backbone.Router.extend(
   initDisplayRouteChanges: (selector) ->
     selector = if selector == undefined then '[href="#' + Backbone.history.fragment + '"]' else selector
     $('nav li.active').removeClass 'active'
-    $('nav li:has(a' + selector + ')').addClass 'active'
-    openMenu menu_speed, openedSign
+    lis = $('nav li:has(a' + selector + ')')
+    lis.addClass 'active'
+    lis.each ->
+      ul = $('ul', this)
+      link = $(this).children('a').first()
+      if link.data('subtree')
+        $.ajax
+          url: link.data('subtree')
+          type: "GET"
+          async: false
```

Comment:
* Duplicating a call to a function is actually the reason we use functions, so we don't duplicate multiple lines
* I'm not sure if we're talking about sibling menu items or cascade calls, but it's always possible to trigger an event when all calls are done

I don't exactly know what is safeness in this context but freezing the JS execution doesn't sound good.
The reason of this change was to avoid page freezes because we have some huge nodes in the media library, so the node rendering will be long, so the ajax request itself will take some time. In the end, the issue will remain.